src/strings/PowerFxResources.en-US.resx

```diff
@@ -4026,6 +4026,60 @@
   <data name="ErrorResource_SuggestRemoteExecutionHint_InOpInvalidColumn_Link_2_URL" xml:space="preserve">
     <value>https://go.microsoft.com/fwlink/?linkid=2132702</value>
     <comment>{Locked}</comment>
+  </data>
+    <data name="ErrorResource_SuggestRemoteExecutionHint_NF_ShortMessage" xml:space="preserve">
+    <value>Delegation warning. The named formula {0} is not delegatable so its usage in this formula might not work correctly on large data sets. </value>
```

Comment:
>delegatable [](http://example.com/codeflow?start=60&length=11)

We use the term 'delegable' instead of 'delegatable' elsewhere, we should be consistent. Personally I find 'delegatable' better, but I'm guessing the decision was made before to go with 'delegable' :-)